app/src/main/java/org/oppia/app/topic/review/TopicReviewViewModel.kt

```diff
@@ -1,7 +1,66 @@
 package org.oppia.app.topic.review
 
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.Transformations
 import androidx.lifecycle.ViewModel
-import org.oppia.app.viewmodel.ObservableViewModel
+import org.oppia.app.fragment.FragmentScope
+import org.oppia.app.model.Profile
+import org.oppia.app.model.ProfileId
+import org.oppia.app.model.Topic
+import org.oppia.app.topic.review.reviewitemviewmodel.TopicReviewItemViewModel
+import org.oppia.domain.topic.TopicController
+import org.oppia.util.data.AsyncResult
+import org.oppia.util.logging.Logger
+import javax.inject.Inject
 
-/** [ViewModel] for showing a list of subtopics. */
-abstract class TopicReviewViewModel : ObservableViewModel()
+/** [ViewModel] for [TopicReviewFragment] */
```

Comment:
In this file and at other places also you are writing subtopic as `subTopic`. We are treating this as a single word so please correct this everywhere.